environment/src/main/resources/schema/app/20210407150154_CB-11874:_Make_DiskEncryptionSetId_available_in_environment_service_API_response.sql

```diff
@@ -0,0 +1,10 @@
+-- // CB-11874: Make DiskEncryptionSetId available in environment service API response
+-- Migration SQL that makes the change goes here.
```

Comment:
To be aligned with the changes of `AzureParameters.java`, please add/remove the new column `encryption_key_url` as well.